manifests/init.pp

```diff
@@ -182,7 +182,7 @@
 # $environment::                            Default environment of the Puppet agent
 #
 # $agent_additional_settings::              A hash of additional agent settings.
-#                                           Example: {stringify_facts => true}
+#                                           Example: {stringify_facts                          => true}
```

Comment:
Could you revert this change and other whitespace only changes?

Reply:
Done! Sorry! :/ Are the whitespace changes in server.pp ok?

Reply:
Yes, indenting is no problem.